deploy/template.yaml

```
@@ -545,6 +545,8 @@ Resources:
                 - 'kms:GenerateDataKey'
               Resource:
                 - !ImportValue AuditEventQueueEncryptionKeyArn
+        - SSMParameterReadPolicy:
+            ParameterName: !Sub ${Environment}/core/coreVtmClaim
```

I think the param name is slightly wrong here. You are missing the "self" bit in the middle.

```suggestion
            ParameterName: !Sub ${Environment}/core/self/coreVtmClaim
```